docs/content/latest/yugabyte-cloud/cloud-secure-clusters/cloud-authentication.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Encryption in transit
+linkTitle: Encryption in transit
+description: Yugabyte Cloud cluster authentication and encryption.
+headcontent:
+image: /images/section_icons/deploy/enterprise.png
+menu:
+  latest:
+    identifier: cloud-authentication
+    parent: cloud-secure-clusters
+    weight: 250
+isTocNested: true
+showAsideToc: true
+---
+
+Yugabyte Cloud uses TLS 1.3 for communicating with clusters, and digital certificates to verify the identity of clusters. Using TLS with a digital certificate encrypts the data in transit to ensure a secure connection to your cluster, and prevents man in the middle (MITM) attacks, impersonation attacks, and eavesdropping.
+
+Whether you are connecting to your cluster from a client shell or an application, Yugabyte Cloud clusters require a TLS/SSL connection.
+
+## Download your cluster certificate
+
+Yugabyte Cloud clusters are provisioned with a digital certificate, which is used to verify the identity of the cluster when you connect to it from an application or client.
+
+Connections using YCQL require the use of the certificate. You can connect to a cluster using YSQL without a certificate, although where possible, the certificate should be used.
+
+You can download the certificate and install it on the computer that will be connecting to the cluster as follows:
```

Comment:
You need to download certificate chain(intermediate CA and root CA) and install ...